projects/1.16/assets/tropicraft/tropicraft/lang/zh_cn.json

```
@@ -0,0 +1,496 @@
+  "item.tropicraft.mai_tai": "迈泰",
+  "item.tropicraft.man_o_war_spawn_egg": "僧帽水母刷怪蛋",
+  "item.tropicraft.manganese_ingot": "锰锭",
+  "item.tropicraft.marlin_spawn_egg": "马林鱼刷怪蛋",
+  "item.tropicraft.music_disc_buried_treasure": "音乐唱片",
+  "item.tropicraft.music_disc_buried_treasure.desc.0": "Punchaface - Buried Treasure",
+  "item.tropicraft.music_disc_buried_treasure.desc.1": "https://www.youtube.com/watch?v=lPhHWYnUR_4",
+  "item.tropicraft.music_disc_eastern_isles": "音乐唱片",
+  "item.tropicraft.music_disc_eastern_isles.desc.0": "Frox - Eastern Isles",
+  "item.tropicraft.music_disc_eastern_isles.desc.1": "https://soundcloud.com/froxlab/eastern_isles",
+  "item.tropicraft.music_disc_low_tide": "音乐唱片",
+  "item.tropicraft.music_disc_low_tide.desc.0": "Punchaface - Low Tide",
+  "item.tropicraft.music_disc_low_tide.desc.1": "https://youtu.be/VgB2TjwGOuQ",
+  "item.tropicraft.music_disc_summering": "音乐唱片",
+  "item.tropicraft.music_disc_summering.desc.0": "Billy Christiansen - Summering",
+  "item.tropicraft.music_disc_summering.desc.1": "https://www.youtube.com/watch?v=szO30RmM7b8",
+  "item.tropicraft.music_disc_the_tribe": "音乐唱片",
+  "item.tropicraft.music_disc_the_tribe.desc.0": "Emile Van Krieken - The Tribe",
+  "item.tropicraft.music_disc_the_tribe.desc.1": "https://www.youtube.com/watch?v=LvxVDhMvwE4",
+  "item.tropicraft.music_disc_trade_winds": "音乐唱片",
+  "item.tropicraft.music_disc_trade_winds.desc.0": "Frox - Trade Winds",
+  "item.tropicraft.music_disc_trade_winds.desc.1": "https://soundcloud.com/froxlab/breakset-37-unnamed",
+  "item.tropicraft.nigel_stache": "奈杰尔的小胡子",
+  "item.tropicraft.orange": "橙",
+  "item.tropicraft.orange_beach_float": "橙色沙滩浮排",
+  "item.tropicraft.orange_chair": "橙色椅子",
+  "item.tropicraft.orange_umbrella": "橙色伞",
+  "item.tropicraft.orangeade": "橙汁",
+  "item.tropicraft.pab_shell": "Pab贝壳",
+  "item.tropicraft.pianguas": "扇贝",
+  "item.tropicraft.pina_colada": "凤梨可乐达",
+  "item.tropicraft.pineapple_cubes": "菠萝果粒",
+  "item.tropicraft.pink_beach_float": "粉红色沙滩浮排",
+  "item.tropicraft.pink_chair": "粉红色椅子",
+  "item.tropicraft.pink_pony_bottle": "粉红色便携式气瓶",
+  "item.tropicraft.pink_scuba_flippers": "粉红色潜水脚蹼",
+  "item.tropicraft.pink_scuba_goggles": "粉红色潜水镜",
+  "item.tropicraft.pink_scuba_harness": "粉红色潜水背带",
+  "item.tropicraft.pink_umbrella": "粉红色伞",
+  "item.tropicraft.piranha_bucket": "食人鱼桶",
+  "item.tropicraft.piranha_spawn_egg": "食人鱼刷怪蛋",
+  "item.tropicraft.poison_frog_skin": "毒蛙皮",
+  "item.tropicraft.purple_beach_float": "紫色沙滩浮排",
+  "item.tropicraft.purple_chair": "紫色椅子",
+  "item.tropicraft.purple_umbrella": "紫色伞",
+  "item.tropicraft.raw_coffee_bean": "生咖啡豆",
+  "item.tropicraft.raw_fish": "生鱼",
+  "item.tropicraft.raw_ray": "生鳐鱼",
+  "item.tropicraft.red_beach_float": "红色沙滩浮排",
+  "item.tropicraft.red_chair": "红色椅子",
+  "item.tropicraft.red_umbrella": "红色伞",
+  "item.tropicraft.roasted_coffee_bean": "烤咖啡豆",
+  "item.tropicraft.rube_nautilus": "鲁布鹦鹉螺",
```

这里是否需要和上方的`Frox`和`Pab`一样保留不译
这些名字是赞助者吗

---

从未设想的原因（）
此处均因不明所义保留原始中文翻译，经查证均为作者名字，故保留不译
```suggestion
  "item.tropicraft.rube_nautilus": "Rube鹦鹉螺",
```